package.json

```diff
@@ -70,11 +70,17 @@
     "deploy-configure": "hardhat deploy --network",
     "docs": "hardhat docgen",
     "format": "npm run format:sol && npm run format:ts",
-    "format:sol": "prettier -c -w --no-editorconfig contracts/**/*.sol contracts/**/**/*.sol",
-    "format:ts": "prettier -c -w --no-editorconfig **/*.ts",
+    "format:fix": "npm run format -- --write",
+    "format:fix:sol": "npm run format:sol -- --write",
+    "format:fix:ts": "npm run format:ts -- --write",
+    "format:sol": "prettier --list-different --no-editorconfig \"{,contracts/**/}*.sol\"",
+    "format:ts": "prettier --list-different  --no-editorconfig \"{,test/**/}*.ts\" \"{,scripts/**/}*.ts\" index.ts",
     "lint": "npm run lint:sol && npm run lint:ts",
-    "lint:sol": "solhint --fix 'contracts/**/*.sol'",
-    "lint:ts": "eslint test --ext .ts --fix",
+    "lint:fix": "npm run lint -- --fix",
```

Comment:
I don't think this works as expected because the `--fix` will be used in the part of the command only and the following command will be run:
```
npm run lint:sol && npm run lint:ts --fix
```

Reply:
let me check it, I tried but I did not realize if it was applied to both

Reply:
updated